.github/ISSUE_TEMPLATE/01-bug-report.yml

```diff
@@ -0,0 +1,74 @@
+name: 🐞 Bug report
+description: Report a problem to help improve this project
+title: "[BUG] <Please write a comprehensive title after the '[BUG] ' prefix>"
```

Comment:
How I understand is, the title will then be "[BUG] Title of the issue". Is this intended? Numpy style would be "BUG: Title of the issue". I'm fine with both.

Reply:
maybe, but @mberz started this style, https://github.com/pyfar/pyrato/issues
both would work for me